SI/Sem.05/Pract.05/Task01.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+int* FindRepeatingElement(int* numbers, int length);
+
+int main()
+{
+	int numbers[] = {1,2,3,0,2,3,6,55,7,1};
+	//int numbers[] = {1,2,3,4,5,6,7,8,9,10};
+	int* answer = FindRepeatingElement(numbers, 10);
+	if (answer[0] == 0)
+	{
+		std::cout << "No repeating elements!" << std::endl;
+	}
+	else
+	{
+		std::cout << "The first repeating element is " <<answer[1]<< std::endl;
+	}
+}
+int* FindRepeatingElement(int* numbers, int length)
```

Comment:
You should return a number. Suggestion - return intex of the first return repeating number. Return -1 if there is no repeating number.